specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2025-05-01/ResourceHealth.json

```diff
@@ -1501,9 +1504,19 @@
           "type": "string",
           "description": "It provides the Timestamp for the given update for the service health event.",
           "format": "date-time"
+        },
+        "eventTags": {
+          "$ref": "#/definitions/eventTags"
         }
       }
     },
+    "eventTags": {
+      "type": "array",
+      "description": "A list of metadata tags associated with the event. Possible values include:\n -Action Recommended: Action may be required by you to avoid possible disruptions or mitigate risks for your services. It is recommended to evaluate these actions and the potential impact on your services. \n- False Positive: After investigation, we've determined your service is healthy and service issues did not impact your services as originally communicated.\n- Preliminary PIR: For our largest, most impactful service issues a Preliminary Post Incident Review (PIR) is published generally within 72 hours of mitigation, to summarize what we have learned so far from the still-in-progress investigation.\n- Final PIR: For service issues, a Final Post Incident Review (PIR) may be published to provide additional details or learnings. Sometimes this requires us to complete an internal retrospective, generally within 14 days of mitigation.",
```

Comment:
this would mean updating swagger for new eventTags. not sure if this is right approach. do we give definitions of possible values in our metadata api response aswell